README.md

```diff
@@ -52,6 +52,11 @@ Then install what you just cloned.
 
 `pip install -e <path-to-platoon-folder>`
 
+### Attention
+Install ordering for dependencies:
+
+1. [NCCL](https://github.com/NVIDIA/nccl) should be installed first.
+2. [libgpuarray/ pygpu](https://github.com/Theano/libgpuarray) should be installed after the installation of NCCL.
```

Comment:
Why do you think this is the case?  libgpuarray will pick up NCCL whenever it is installed because it dynamically loads it.

Or maybe you are using an old version which would only detect at install time.